plugins/org.eclipse.elk.alg.layered/src/org/eclipse/elk/alg/layered/Layered.melk

```diff
@@ -87,53 +87,53 @@ algorithm layered(LayeredLayoutProvider) {
     // splines
     supports edgeRouting.splines.mode
     supports edgeRouting.splines.sloppy.layerSpacingFactor
-	// other options
-	supports org.eclipse.elk.padding = new ElkPadding(12)
-	supports org.eclipse.elk.edgeRouting = EdgeRouting.ORTHOGONAL
-	supports org.eclipse.elk.port.borderOffset = 0
-	supports org.eclipse.elk.randomSeed = 1
-	supports org.eclipse.elk.aspectRatio = 1.6f
-	supports org.eclipse.elk.noLayout
-	supports org.eclipse.elk.portConstraints
-	supports org.eclipse.elk.port.side
-	supports org.eclipse.elk.debugMode
-	supports org.eclipse.elk.alignment
-	supports org.eclipse.elk.hierarchyHandling
-	supports org.eclipse.elk.separateConnectedComponents = true
-	supports org.eclipse.elk.insideSelfLoops.activate
-	supports org.eclipse.elk.insideSelfLoops.yo
-	supports org.eclipse.elk.nodeSize.constraints
-	supports org.eclipse.elk.nodeSize.options
-	supports org.eclipse.elk.direction = Direction.UNDEFINED
-	supports org.eclipse.elk.nodeLabels.placement
-	supports org.eclipse.elk.nodeLabels.padding
-	supports org.eclipse.elk.portLabels.placement
+    // other options
```

Comment:
I'm not sure if it's only due to these changes, but the indentation of the melk files is somewhat broken and inconsistent now. 
Can you please fix this?